pkg/apis/agones/v1/gameserver.go

```diff
@@ -150,6 +151,20 @@ type GameServerSpec struct {
 	SdkServer SdkServer `json:"sdkServer,omitempty"`
 	// Template describes the Pod that will be created for the GameServer
 	Template corev1.PodTemplateSpec `json:"template"`
+	// AlphaSpec describes the alpha properties for the GameServer
```

Comment:
nit: The variable is actually "Alpha". AlphaSpec is the type.

Reply:
Nice catch!